rpcs3/Emu/Cell/PPUModule.cpp

```diff
@@ -333,7 +333,7 @@ static void ppu_initialize_modules(ppu_linkage_info* link)
 
 				if (!next || (end >> 16 != alloc_addr >> 16))
 				{
-					alloc_addr = vm::alloc(0x10000, vm::main);
+					alloc_addr = vm::alloc(SYS_PROCESS_PARAM_MALLOC_PAGE_SIZE_64K, vm::main);
```

Comment:
True they have the same value, but this is not the meaning of the enum.

Reply:
yeah. those were the ones I wasn't sure about